frontend/src/themes/dark-theme.ts

```diff
@@ -278,6 +278,19 @@ const theme = {
             // A700: '#A6000E',
         },
         variants: colors.variants,
+
+        /**
+         * Dashboard and charts
+         */
+        charts: {
+            gauge: {
+                gradientStart: '#4C4992',
+                gradientEnd: '#9792ED',
+                background: '#39384C',
+                sectionLine: '#8c89bf',
+                text: colors.grey[800],
```

Comment:
These should be coming from the theme? Or align with Nicolae which theme colors to use instead.

Reply:
That's the approach in entire dark theme, unfortunately

Reply:
I aligned with Nicolae. We already have some of the colors used in theme, but in different context, and I didn't want to mix it. Using `palette.secondary.main` next to `palette.spotlight.border` is confusing. I don't want other components to influence colors on this chart.

Reply:
We don't want hard coded colors. Period. So we need to align and add these colors to theme as more appropriate labels. This is technical debt accumulation.